.github/workflows/ci.yml

```diff
@@ -11,9 +11,9 @@ jobs:
       matrix:
         python-version: ["3.9", "3.10", "3.11"]
     steps:
-      - uses: actions/checkout@v2
+      - uses: actions/checkout@v3
       - name: Set up Python
-        uses: actions/setup-python@v2
+        uses: actions/setup-python@v4
```

Comment:
action went up two versions?

Reply:
I originally wrote this code using GitHub copilot, it generated code with the v2 suggestion

Reply:
v3 https://github.com/actions/setup-python/commit/8c1441c0076e195efc99cca48e57b56d6c9713ac was for https://github.com/actions/setup-python/commit/93cb78f17ba30b733a6c17d0f21183bdc0140887 (node16)
v4 https://github.com/actions/setup-python/commit/775367df99e790d68426e89b22fb536e5c0fa248 was for https://github.com/actions/setup-python/commit/53e15292cd6a522cb460037b6f031eadbd6f6920 (python version file)